jaworski_cellseg/load_data_widget.py

```diff
@@ -2,19 +2,83 @@
 import napari
 from pathlib import Path
 from bioio import BioImage
+import tifffile
+import numpy as np
 
 
-def jwslab_load_bio_data(napari_viewer: napari.Viewer, file_path: Path):
-    print("enter jwslab_pre_process_file")
+def jwslab_load_bio_data(
+    napari_viewer: napari.Viewer, file_path: Path, physical_sizes: dict
+):
     try:
```

Comment:
It's best practice to include the error prone code only in `try` blocks (easier to debug later). It seems like what might go wrong is that a bad file path was provided. Consider creating two helper functions, one for tiff-like files and another for `[".oir", ".nd2"]` and handle the exceptions within them.

Reply:
Done